docs/source/testing-and-debugging.md

```diff
@@ -135,3 +135,37 @@ of the transaction.
         result, _ = await self.call("get_message", return_type=str)
         self.assertEqual(new_message, result)
 ```
+
+#### Accessing the Storage
+
+To get the key-value pairs stored in the smart contract's storage, you can use the `get_storage` method from 
+`SmartContractTestCase`. Use it with the prefix of the keys you want to access, and you'll get a `dict[bytes, bytes]`.
+If you want to get a dictionary of other types, you can also pass a function as argument to process the keys or values
+before adding it to the dict. If want to remove the prefix from the result you can use the `remove_prefix` parameter.
+
+```python
+    # inside the HelloWorldWithDeployTest class
```

Comment:
this is an extension of the HelloWorld contract test case isn't it?
if so, can you change this example to access the HelloWorld storage instead of GASToken? it uses storage without prefixes, so I think it's more intuitive for an example.